src/opower/utilities/base.py

```diff
@@ -12,7 +12,8 @@ class UtilityBase:
     def __init_subclass__(cls, **kwargs) -> None:
         """Keep track of all subclass implementations."""
         super().__init_subclass__(**kwargs)
-        cls.subclasses.append(cls)
+        if not cls.__name__.startswith("_"):

```

Comment:
Instead of this check, how about the evergy base class to not implement the UtilityBase? See the Exelon class as an example.